spec/beaker/hypervisor/openstack_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'fog'
+
+module Beaker
+  describe OpenStack do
+
+    let(:options) { make_opts.merge({'logger' => double().as_null_object}) }
+
+    let(:openstack) {
+      stub_const("Fog::Compute", MockComputeClient)
+      stub_const("Fog::Network", MockNetworkClient)
```

Comment:
@azhurbilo is there a reason that you need two mock files for these, rather than just creating a few stubbed objects in this file?